examples/README.md

```diff
@@ -12,9 +12,9 @@ This directory contains examples of Bokeh Apps, which are simple and easy to cre
 
 ### [`charts`](https://github.com/bokeh/bokeh/tree/master/examples/compat)
```

Comment:
I just noticed in the full version of this readme (shakes fist at diffs) this note 

> "NOTE: All of the links below are to the "master" branch of the repository, which always contains the newest and most updated code. For questions about specific versions of Bokeh, please take care to refer to the correct branch for your version!"  

I would also propose removing that note I guess. Or reframing it as make sure the version of the examples matches the version of bokeh you're running.